### ISSUE TITLE
Indexer Request: Extra Torrent

### ISSUE THREAD

**Issue by markj24** [cardigann/cardigann#206]
### English

**Comment by alberto467** [comment]
#227 I've been trying: https://gist.github.com/alberto467/e9db2f5480753725bd5589d2be33cecd

**Comment by zductiv** [comment]
@alberto467 Link is dead. Do you still have this one?

**Comment by alberto467** [comment]
@zductiv Yes,  I took down the link because I've been having some problems: ExtraTorrent has been obfuscating the torrent table with js, so you can't get any data at all off of the site. They've been turning this on and off so sometimes it works perfectly, sometimes  it doesn't work at all. (Check issue #281). Unfortunately my pc's psu failed so I can't really do anything until it comes back from the warranty.
